docs/versions/latest/Components/Archetypes/typescript/Components/Workflows.md

```diff
@@ -366,3 +364,91 @@ export class HandleNetworkConfigurationBackup {
   }
 }
 ```
+
+### `@UserInteractionWorkflowItem`
+
+The decorator is used to specify an user interaction workflow item.
+
+#### Supported Parameters
```

Comment:
Maybe we should specify how to configure the form? It's following:

{WORKFLOW_NAME}_{USER_INTERACTION_ACTION}.form.json, correct?